lib/elixir/lib/io.ex

```diff
@@ -567,6 +567,7 @@ defmodule IO do
 
   """
   @doc since: "1.12.0"
+  @spec stream() :: Enumerable.t()
```

Comment:
We now have the `t:Enumerable.t/1` type, so I would use this here:

```suggestion
  @spec stream() :: Enumerable.t(String.t())
```